src/core/inlines.js

```diff
@@ -118,6 +130,7 @@ export function run(conf) {
       "\\b(?:NOT\\s+)?RECOMMENDED\\b",
       "\\bOPTIONAL\\b",
       "(?:{{3}\\s*.*\\s*}{3})", // inline IDL references,
+      "\\B\\|\\w+(?:\\s*\\w+)*(?:\\:\\s*\\w+)?\\|\\B", // inline variable regex
```

Comment:
```suggestion
      "\\B\\|\\w[\\s\\w]*\\w(?:\\s*\\:\\s*\\w+)?\\|\\B", // inline variable regex
```

Reply:
I was being lazy and use commit suggestions to edit two files from github.com UI